src/main/java/org/cascadebot/cascadebot/CascadeBot.java

```diff
@@ -91,7 +96,12 @@ public static void main(String[] args) {
         if (System.getenv("SENTRY_DSN") == null) {
             LOGGER.warn("You haven't set a Sentry DNS in the environment variables! Set SENTRY_DSN to your DSN for this to work!");
         }
-        INS.init();
+        try {
+            INS.init();
+        } catch (Exception e) {
+            LOGGER.error("Error on initialisation!", e);
+            ShutdownHandler.exitWithError();
```

Comment:
is this the one that tells the wrapper not to restart the bot?

Reply:
Yup